implementations/rust/ockam/ockam_command/src/identity/create.rs

```diff
@@ -26,7 +27,7 @@ async fn run_impl(
     let mut rpc = Rpc::background(&ctx, &options, &cmd.node_opts.api_node)?;
     let request = Request::post("/node/identity");
     rpc.request(request).await?;
-    rpc.parse_response()?;
+    rpc.parse_response::<CreateIdentityResponse>()?;
 
     println!("Identity created!");
```

Comment:
In a previous PR, we lost some information here. It should be:

```rust
res = rpc.parse_response::<CreateIdentityResponse>()?;
println!("Identity {} created!", res.identity_id)
```

Could you add it back while we are here?

Reply:
Done. Thanks.